app/src/main/res/menu/action_bar_menu.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/search_go_btn"
+        android:checked="true"
+        android:icon="@drawable/ic_search"
+        android:title="Search"
+        app:showAsAction="ifRoom">
+
+    </item>
+    <item
+        android:id="@+id/home"
+        android:title="Home"
+        app:showAsAction="never">
+
+    </item>
+    <item
+        android:id="@+id/favorite"
```

Comment:
3.2 Naming resources

Resource IDs and names need to be declared in lowercase_underscore

3.2.1 Naming ID

ID should be prefixed with name of element and use underscore. For example :
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md